nasa-jpl/explorer-1
# Proof of concept: Explorer 1 themes

### Checklist

  - PR titles are used to generate the change log in [releases](../releases); good ones make that easier to scan.
  - PRs will be broadly categorized in the change log, but for even easier scanning, consider prefixing with a component name or other useful categorization, e.g., "BaseButton: fix layout bug", or "Storybook: Update dependencies".
  - These tags are used to do the aforementioned broad categorization in the change log and determine what the next release's version number should be.
  - [Release Drafter](https://github.com/marketplace/actions/release-drafter) will attempt to do the category labeling for you! Please double-check its work.

## Description

Adds Tailwindcss themer and introduces a few semantic color aliases per https://github.com/nasa-jpl/www/issues/19#issuecomment-2125971990

## Instructions to test

You must have pnpm installed.

1. `make install`
2. `make vue`
3. View http://localhost:5173/
4. Edit `/packages/vue/index.html` and change `<html data-theme="defaultTheme">` to `<html data-theme="edu">`
5. Save and check the browser. You should now see edu colors.



## Tested in the following environments/browsers:



### Operating System


### Browser
